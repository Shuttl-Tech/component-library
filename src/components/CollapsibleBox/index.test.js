import React from 'react';
import renderer from 'react-test-renderer';

import { CollapsibleBox } from './';

it('CollapsibleBox tree looks like how it should', () => {
	let props = {
		match: {
			params: {
				id: 1
			}
		}
	};

	const component = renderer.create(<CollapsibleBox {...props} />);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
