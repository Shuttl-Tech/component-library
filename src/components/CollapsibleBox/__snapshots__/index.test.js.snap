// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollapsibleBox tree looks like how it should 1`] = `
<div
  className="box component--box container container collapsed-partially component--collapsible-box--container--collapsed-partially"
>
  
  <div
    className="body box-module component--modular-box--body component--modular-box--box-module body collapsed-partially component--collapsible-box--container--collapsed-partially component--collapsible-box--body"
  />
  <div
    className="footer box-module component--modular-box--footer component--modular-box--box-module"
  >
    <div
      className="component--collapsible-box--footer"
      onClick={[Function]}
    />
  </div>
</div>
`;
